guides/common/modules/proc_configuring-repositories-proxy.adoc

```diff
@@ -11,7 +11,6 @@ Select the option that corresponds with operating system and version you want to
 
 ifdef::foreman-el,katello,satellite[]
 * xref:#repositories-rhel-8[{RHEL} 8]
```

Comment:
If there's just a single entry, does it make sense to keep this xref in?

Reply:
Removed :+1: